corporate_it_service/models/applicant.py

```diff
@@ -0,0 +1,184 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from datetime import date
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+available_priority = [
+    ("0", "Normal"),
+    ("1", "Good"),
+    ("2", "Very Good"),
+    ("3", "Excellent"),
+]
+
+
+# create a class for a applicant
+class JobApplicant(models.Model):
+    _name = "it.applicant"
+    _description = "Job Applicant"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+
+    # fields for applicant
+    name = fields.Char(
+        string="Applicant Id",
+        required=True,
+        copy=False,
+        readonly=True,
+        default=lambda self: _("New"),
+    )
+    boolean = fields.Boolean(string="Active", default=True, required=True)
+    applicant_id = fields.Char(string="Applicant Name", required=True)
+    mobile = fields.Char(string="Applicant Mobile", store=True)
+    applicant_email = fields.Char(
+        string="Email", help="Applicant Email", required=True, store="True"
+    )
+    applicant_degree = fields.Selection(
+        [
+            ("bba", "BBA"),
+            ("mba", "MBA"),
+            ("bca", "BCA"),
+            ("bsc", "BSC"),
+            ("msc", "MSC"),
+            ("b.tech", "B.TECH"),
+            ("m.tech", "M.TECH"),
+            ("phd", "PHD"),
+        ],
+        string="Highest Degree",
+    )
+    applicant_date = fields.Date(
+        string="Application Date", readonly=True, default=fields.datetime.now()
+    )
+    availability = fields.Date(string="Availability of Applicant", required=True)
+
+    salary_expect = fields.Float(
+        string="Salary Expectation",
+        store=True,
+        required=True,
+    )
+    platform_via = fields.Selection(
+        [
+            ("facebook", "FACEBOOK"),
+            ("linkedin", "LINKEDIN"),
+            ("website", "OUR WEBSIDE"),
+            ("google", "GOOGLE"),
+        ],
+        string="Which Plateform to find job",
```

Comment:
The string should be the job platform and "Which Platform to find a job" should be in the help text. Also, fix the typo of  Plateform to the platform